internal/nodes/avi_model_l4_translator.go

```diff
@@ -451,14 +451,113 @@ func PopulateServersForNodePort(poolNode *AviPoolNode, ns string, serviceName st
 				}
 
 			}
+			node4Found := true
+			node6Found := true
+			if svcObj.Spec.ExternalTrafficPolicy == v1.ServiceExternalTrafficPolicyLocal {
```

Comment:
Will this logic kicked in if POD goes down on specific node and we need to remove endpoint and add different one?